fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -918,8 +917,20 @@ public int getBlockSize() {
         return blockSize;
     }
 
-    public FDBRecordContext getContext() {
-        return context;
+    @VisibleForTesting
+    FDBRecordContext getCallerContext() {
+        // We cannot control the way the returned context is used. Limiting it to testing seems to be a good idea.
```

Comment:
It would probably be good to followup with removing this too, but that can probably wait.